### ISSUE TITLE
Git-for-windows 2.47.1 x86_64 does not work on windows 7 (`GetSystemTimePreciseAsFileTime` not found in dll `kernel32.dll`)

### ISSUE THREAD

**Issue by kralo** [git-for-windows/git#5280]
Git-for-windows 2.47.1 x86_64 does not work on windows 7 (git bash cannot be started), possibly due to an error in mintty, reported there (`GetSystemTimePreciseAsFileTime` not found in dll `kernel32.dll`), just so you are aware.

https://github.com/mintty/mintty/issues/1298

**Comment by kralo** [comment]
If you don't want to run on windows 7 

> Git for Windows for Windows v2.47 drops support for Windows 7 and for Windows 8, as announced previously.

then maybe just don't allow installation or create an error upon install?

**Comment by KoleckOLP** [comment]
The website should not say windows 7 SP 1 is supported than

https://gitforwindows.org/requirements.htmln.

**Comment by dscho** [comment]
> The website should not say windows 7 SP 1 is supported than.
> 
> https://gitforwindows.org/requirements.html

Good point. @KoleckOLP could you kindly help out and open a PR at https://github.com/git-for-windows/git-for-windows.github.io?
